docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1879,6 +1879,48 @@ Current number of open HTTP connections for the node.
 `total_opened`::
 (integer)
 Total number of HTTP connections opened for the node.
+
+`clients`::
+(array of objects)
+Information on current and recently-closed HTTP client connections.
++
+.Properties of `clients`
+[%collapsible%open]
+=======
+`id`::
+(integer)
+Unique ID for the HTTP client.
+`agent`::
+(string)
+Reported agent for the HTTP client, if available.
+`local_address`::
+(string)
+Local address for the HTTP client.
+`remote_address`::
+(string)
+Remote address for the HTTP client.
+`forwarded_for`::
```

Comment:
It looks like the `last_uri` property may be missing:
https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/http/HttpStats.java#L74

Reply:
Thanks, @jrodewig! I missed a bunch of stuff on this one. I add the missing `last_uri` stat in https://github.com/elastic/elasticsearch/pull/70512/commits/aa3efd4d60893711575fa91633ae6915bf580b41.